tests/ui/crashes/ice-6251.rs

```diff
@@ -1,5 +1,5 @@
 // originally from glacier/fixed/77329.rs
-// assertion failed: `(left == right) ; different DefIds
+// assertion failed: `(left == right) ; different DefIds`
```

Comment:
Should be
```suggestion
// assertion failed: `(left == right)` ; different DefIds`
```
This message has been updated after this test was added, but it doesn't really matter